sfdc/modules/bulk/utils.bal

```diff
@@ -24,6 +24,8 @@ import ballerina/lang.'xml as xmllib;
 import ballerina/regex;
 import ballerinax/sfdc;
 
+isolated string csvContent = "";
```

Comment:
Can you check other places also? Similar thing is repeated in utils.bal
```suggestion
isolated string csvContent = EMPTY_STRING;
```

Reply:
Fixed in 2ea4f30e2ed08309e52e4026d558a483d059f7a6 . Thanks